Bottom: 92aa412637fc967fdce9639834d9e13b642adacf
Top:    eabdc8f6e59ec1e21f07cdb696f77768108cb52e
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-19 20:17:34 +0800

Refresh of drop-vhost_copy_from

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index ff2f214..02049d2 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -2023,9 +2023,6 @@ int vhost_prefetch_desc_indices(struct vhost_virtqueue *vq,
 		++ret;
 		++last_avail_idx;
 	}
-
-	vq->last_avail_idx = last_avail_idx;
-
 	/* FIXME: update used ring here? together with batch dequing? */
 
 	/* Only get avail ring entries after they have been exposed by guest. */
